src/model/db_template.py

```diff
@@ -0,0 +1,48 @@
+"""Copyright by CookieDecide, Darkuuu
+Licensed under MIT License
+"""
+
+import peewee as pw
+import os
+
+# check if db folder exists
+if not os.path.exists("../db"):
+    os.mkdir("../db")
+
+# link the database file
+TEMPLATE_DB = pw.SqliteDatabase("../db/template.db", check_same_thread=False)
```

Comment:
Does this make sense to execute this code always when this module is imported? Wouldn't it be better to implement a Singleton (or does pw take care of doing that)?

Additionally, I would make the string "../db/template.db" as a field on top of that class to make it easier to change.

Reply:
How would you go about implementing that as a singleton?

Reply:
Clarified in call. Will go with this approach for now, if problematic will be fixed later.